javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+var $header = document.querySelector('h1');
+var counter = 4;
+
+setInterval(function () {
+  if (counter > 1) {
+    counter--;
+    $header.textContent = counter;
+  } else {
+    $header.textContent = 'eternal pain and suffering. what is life';
+    clearInterval();
```

Comment:
This isn't doing anything, you should look at the necessary argument for `clearInterval` in the docs